### DISCUSSION TITLE
42 Bangkok Onsite Cluster Map

### DISCUSSION THREAD

**Discussion by hazamashoken** [42-Bangkok/portal#12] [Ideas]
Possible features 

- cluster map to show tables/cluster on 42Bangkok site
- populate the map with people log-in onsite

Possible bugs regarding 42 API
there maybe multiple ID on a single table

There's is no direct way to filter who is on site rn.

The current possible solution is to use `/v2/campus/33/users?sort=-updated_at,status&status=1&page[size]=100&page[number]=1`
and loop through to the end then filter profile with location field value
`"location": "z3t4c1"`,

if we can filter `active?` or `location` fields directly it would reduce the amount of request to the backend

**Comment by kcp2p** [comment]
> Possible bugs regarding 42 API there maybe multiple ID on a single table

Probably a problem from our side… try to find out which table have multiple users and report to me.
